### ISSUE TITLE
GPU requirements for training

### ISSUE THREAD

**Issue by francescobodria** [HuguesTHOMAS/KPConv-PyTorch#254]
Hi I'm trying to use your repository to train on a custom dataset
Do you have any recommendation for memory? RAM and VRAM?

**Comment by TokyoWarfare** [comment]
ok I've just started doing training.

surprisingly my 1080 was doing right. And I noticed a thing, while teh gpu just has 8Gb , windows offers + 32Gb of extra shared memory. It may be slower but OOM bar is moved MUUUUUCH further. 

So far I was being able to train chunks of up to 140k points WITHOUT features. And crashed when trying to train a 170k points chunk, failed for 2gb, may be enabling some mem optimizations could even fit theese.

UNFORTUNATELLY there is no way to further extend the shared mem ammount this is ruled by the Nvidia driver that seems to cap at 32 for consumer cards.... 

While this is superb for the 1080 and other lower range users, it's kind of a boomer for people (like me) who invested on a second pc for training, meaning the 1080 can allocate 8+32=40 GB memory while the 3090ti can 24+32 =56GB, which is not that much, and given the slower memory sets the pace, as said there is difference, but not much, andhaving almost 200GB ram on the new pc, seeint so much underused, hurts.

So, long story short even a 1080 can do training now tx to shared memory, at the cost of lower mem speeds. +you've some metrics on what fits on that ammount of memory so do your maths :D
